elastic/datadog_checks/elastic/metrics.py

```diff
@@ -542,6 +542,7 @@
     'elasticsearch.indices.request_cache.evictions': ('rate', 'indices.request_cache.evictions'),
     'elasticsearch.indices.request_cache.evictions.count': ('monotonic_count', 'indices.request_cache.evictions'),
     'elasticsearch.indices.request_cache.hit_count': ('rate', 'indices.request_cache.hit_count'),
+    'elasticsearch.indices.request_cache.hit_count.count': ('monotonic_count', 'indices.request_cache.hit_count'),
```

Comment:
This is a post 2 metric. It was listed on metadata csv but not actually collected